### PR TITLE
Add pytest command line example to pytest README.md

### PR DESCRIPTION
**Pull request by wangxin**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
Add example and guideline for how to run test scripts using pytest. Also explained the reason of why sonic-mgmt extended pytest options must be after test scripts on command line.

#### How did you do it?
Updated sonic-mgmt/tests/README.md

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -40,3 +40,44 @@ Hosting pytest test infrastructure and test cases
...
+```
+pytest [test script files or directories containing test script files] [options listed in "pytest --help" only] [options extended in sonic-mgmt]
+```
+
+For example:
+```
+pytest test_announce_routes.py --inventory veos_vtb --host-pattern vlab-01 --testbed_file vtestbed.csv --log-cli-level info
+```
+
+Please ensure that the command line options extended in sonic-mgmt are always added **AFTER** the test script files or directories container test script files. Otherwise, pytest may complain that the options defined in `conftest.py` of sub-folders are unrecognized.
```
Comment (neethajohn) [inline tests/README.md:56]: s/container/containing

File: tests/README.md [hunk 2]
```diff
@@ -40,3 +40,44 @@ Hosting pytest test infrastructure and test cases
...
+
+After pre-parse is done, pytest will try to locate all the `conftest.py` files related the discovered test scripts, which is `vms-kvm-t0` in this example. File `sonic-mgmt/tests/conftest.py` will be loaded by pytest because it is in the same directory of `vms-kvm-t0`. However, file `sonic-mgmt/tests/example/conftest.py` is not loaded as expected. Then pytest will parse all the command line arguments and complain that `--example_option1` is unrecognized argument.
+
+If we run the command like below, then there is no problem.
+
+```
+pytest --inventory veos_vtb --host-pattern vlab-01 example/test_example.py --testbed vms-kvm-t0 --testbed_file vtestbed.csv --log-cli-level debug --disable_loganalyzer --skip_sanity --example_option1 my_value1
+```
+
+During pre-parse, pytest recognizes that `example/test_example.py ` is a test script. Then pytest will try to load file `sonic-mgmt/tests/try_opt/conftest.py`. After this conftest.py file is loaded, there is not problem of parsing argument `--example_option1`.
```
Comment (neethajohn) [inline tests/README.md:83]: s/there is not/there is no

Comment (wangxin) [reply tests/README.md]: Thanks Neetha!

**Comment by bingwang-ms** [comment]
Do we need to update run_tests.sh?

**Comment by wangxin** [comment]
@bingwang-ms Yes, need another PR to update run_tests.sh.

**Pull request merged**